docs/get-started.md

```diff
@@ -0,0 +1,103 @@
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in version v1.10.0 or later
+* [svcat CLI](https://svc-cat.io/docs/cli/) in version v1.10.0 or later
+
+## Steps
+
+Follow these steps to configure your own bundles repository and to complete this Getting Started guide.
+
+### Create a fork
+
+1. Fork the [bundle repository](https://github.com/kyma-project/bundles).
+
+2. On your forked repository, create a `gh-pages` branch if it doesn't already exist.
+
+3. Go to the **Settings** tab. In the **GitHub Pages** section, set up **Source** to `gh-pages branch`.
+
+### Integrate Travis CI with the GitHub repository
+
+1. Go to the [GitHub](https://github.com) page.
+
+2. In your GitHub profile account, choose the **Settings** tab. In the left sidebar, click **Developer settings**.
+
+3. In the left sidebar, click **Personal access tokens**.
+
+4. Click **Generate new token**.
+
+5. Give your token a descriptive name.
+
+6. Select the scope of the token `public_repo`.
+
+7. Click **Generate token** and copy your personal access token.
+
+8. Go to [Travis CI](https://travis-ci.com/) and sign up with your GitHub account. Accept the **Authorization** of Travis CI.
+
+9. Click the **Activate** button and select your forked bundles repository. Click the **Approve & Install** button to give Travis CI access to your forked `bundle` repository.
+
+10. Click the **More options** tab on the right and select **Settings**.
+
+11. Click the **More options** tab on the right and select **Settings**.
+
+12. Go to the **Environment Variables** section. Set **Name** as `GITHUB_TOKEN`. **Value** is equal to your GitHub personal access token. Click the **Add** button.
+
+13. Go to the **More options** tab and select **Trigger build**. Select a master branch and click the **Trigger custom build** button.
+
+### Create a bundle
+
+1. Create your bundle according to the [instruction](https://kyma-project.io/docs/latest/components/service-brokers#configuration-how-to-create-a-ybundle)
+
+2. Place your bundle into the `bundle/{repository}` directory.
+
+3. Along your bundle directory create `index.yaml` which defines available bundles. 
+Instruction how to create a valid file can be found [here](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/011-configuration-helm-broker.md#configuring-the-helm-broker-externally)
+
+4. Commit changes and push to the master branch.
+
+5. Travis automatically launches new build. You can check build status on [travis-ci](https://travis-ci.com/) webpage.
+
+### Setup Helm Broker
+
+1. Go to your forked repository and click the **Settings** tab. Go to the **GitHub Pages** section and copy Github pages link.
+  
+2. Set up the environment variable on the Helm Broker Deployment:
+    ```
+    kubectl set env -n kyma-system deployment/core-helm-broker -e APP_REPOSITORY_BASE_URL=https://user-name.github.io/bundles/{repository}/
```

Comment:
In step 1 we are saying about copying the url and here we already adding a pattern: 
`APP_REPOSITORY_BASE_UR=https://user-name.github.io/bundles/{repository}/`
IMO this should be replaced with 
`APP_REPOSITORY_BASE_UR={github_page_link}`
or 
`APP_REPOSITORY_BASE_UR=https://{user-name}.github.io/bundles/{repository}/`